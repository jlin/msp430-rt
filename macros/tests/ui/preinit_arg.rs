#![no_main]

use msp430_rt_macros::{entry, pre_init};

#[entry]
fn bar() -> ! {
    loop {}
}

#[pre_init(arg)]
unsafe fn foo() {}
