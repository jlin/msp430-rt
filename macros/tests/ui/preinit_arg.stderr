error: this attribute accepts no arguments
  --> tests/ui/preinit_arg.rs:10:1
   |
10 | #[pre_init(arg)]
   | ^^^^^^^^^^^^^^^^
   |
   = note: this error originates in the attribute macro `pre_init` (in Nightly builds, run with -Z macro-backtrace for more info)
